crates/node/src/chain_spec.rs

```diff
@@ -8,6 +9,10 @@ use sp_core::storage::Storage;
 use sp_core::{Pair, Public};
 use sp_state_machine::BasicExternalities;
 
+lazy_static! {
+    static ref STARKNET_GENESIS: String = read_file_to_string("crates/node/src/genesis_assets/genesis.json");
+}
```

Comment:
I think we should not have it as a static.
Static means it will stay in memory for the whole execution of the node (potentially months).

From what I see it is only used during the startup of the node:
- to print it
- to initialize the runtime

The startup is not something we try to optimize. Reading it two times from the disk instead of once is not a large overhead and this way we don't have to keep it in memory for the rest of the execution. It is dropped when not used anymore

Reply:
Yep, you are completely right.